src/pages/billing/CreateNewBillingProjectWizard.test.js

```diff
@@ -244,7 +244,7 @@ describe('CreateNewBillingProjectWizard Steps', () => {
       verifyEnabled(getStep2HaveBillingAccountButton())
     })
     it('has the checkbox checked for Step 3', () => {
-      verifyChecked(getStep3CheckBox())
+      verifyChecked(screen.getByRole('checkbox', 'I have verified the user has been added to my account (requires reauthentication)'))
```

Comment:
Why can't you just call getStep3CheckBox here? I was just suggesting you not assume it's the only checkbox, but a helper method is still a good thing to avoid repeating the text everywhere.

Reply:
The tests fails if I use `getStep3CheckBox()` -- I don't think the element that `queryByText ` returns can be used with to` .toBeChecked()`.  This is the error that I get: 
`toBeChecked() can only be used with type="checkbox" or type="radio" or elements with role="checkbox". `
I can try using `toHaveAttribute() ` instead of `toBeChecked() ` in the `verifyChecked` method and call getStep3Checkbox() throughout the test if that makes it easier to read the test.

Reply:
I changed the getStep3Checkbox() helper method to screen.getByRole('checkbox', 'I have verified the user has been added to my account (requires reauthentication)' instead of screen.queryByRole('checkbox')